src/util/types.ts

```diff
@@ -63,12 +63,14 @@ export interface UIOrder {
     filled: BigNumber;
     price: BigNumber;
     status: OrderStatus;
+    emptySize: BigNumber;
 }
 
 export interface OrderBookItem {
     side: UIOrderSide;
     size: BigNumber;
     price: BigNumber;
+    emptySize: BigNumber;
```

Comment:
And here *maybe* we can just use size, since the size of an order book item is meant to be the sum of the available parts of the orders for that price. I'm not 100% sure about this, though.